src/platform/i386/chal/chal_proto.h

```diff
@@ -32,8 +32,7 @@
 typedef struct pgtbl *pgtbl_t;
 
 struct pgtbl_info {
-	asid_t  asid; /* Unused */
-	pgtbl_t pgtbl;
+	pgtbl_t  pgtbl;
```

Comment:
Lets talk soon about how we can fit everything we need. Maybe Monday with @WenyuanShao ?